src/monitor-view-events.js

```diff
@@ -0,0 +1,53 @@
+// DOM Refresh
```

Comment:
I think Renderer and TemplateCache could also potentially go into whatever bucket this belongs in